src/wp-includes/rest-api/endpoints/class-wp-rest-pattern-directory-controller.php

```diff
@@ -119,16 +119,7 @@ public function get_items( $request ) {
 			$query_args['slug'] = $slug;
```

Comment:
Should this be passed through, `wp_parse_list`

Reply:
Good point. Fixed in 0a2c2ebfbfaf9de86be352e0957942f73b253f81

Reply:
It is passed through `wp_parse_list` here: https://github.com/WordPress/wordpress-develop/blob/trunk/src/wp-includes/rest-api/class-wp-rest-server.php#L983
More specifically, here: https://github.com/WordPress/wordpress-develop/blob/trunk/src/wp-includes/rest-api.php#L1500
So, in theory, this is not needed.